delfin/tests/unit/drivers/dell_emc/vnx/vnx_block/test_vnx_block.py

```diff
@@ -331,3 +318,63 @@ def test_err_cli_res_to_list(self):
             navi_handler = NaviHandler(**ACCESS_INFO)
             navi_handler.cli_res_to_list({})
         self.assertIn('cli resource to list error', str(exc.exception))
+
+    @mock.patch.object(time, 'mktime')
+    def test_time_str_to_timestamp(self, mock_mktime):
+        tools = Tools()
+        log_time = '03/26/2021 14:25:36'
```

Comment:
What does `log_time` mean? Please use a meaningful variable name

Reply:
Please make sure that the unit test is useful, for example, what the expected value after we called `time_str_to_timestamp`? Please check that.

Reply:
modifed